### PR TITLE
fix: add select keyboard accessibility adjustments

### PR DESCRIPTION
**Pull request by davidmenendez**
takes a look at some basic keyboard navigation enhancements for add select as well as a few other minor v11 styling fixes. 
this does remove some key "features", namely row focusing styling and displaying hidden functionality until row hover, but i believe these specific interactions are directly responsible for some of the accessibility problems with add select. so if design has a problem i suggest they work with accessibility to find a solution. breadcrumbs for hierarchy mode should now also be accessible via the keyboard.

### CODE REVIEW

**Comment by lee-chase** [comment]
@davidmenendez need to run prettier?

**Comment by davidmenendez** [comment]
@lee-chase fixt

**Comment by davidmenendez** [comment]
@lee-chase checks are fixed now

File: packages/cloud-cognitive/src/components/AddSelect/_add-select.scss
Comment (elycheea) [inline packages/cloud-cognitive/src/components/AddSelect/_add-select.scss:1]: I think you can regain the visibility / hover with something like this ....

```scss
.#{$block-classs}__selections-dropdown {
  visibility: hidden;
}


.#{$block-classs}__selections-row:focus-within .#{$block-classs}__selections-dropdown
.#{$block-classs}--add-select__selections-row:hover .#{$block-classs}__selections-dropdown {
  visibility: visible;
}

```

**Review by elycheea — COMMENTED** [review]
Overall, the interactions actually feel much smoother via keyboard.

I shared a suggestion for a CSS based solution for getting the dropdowns back on hover/focus-within.

There are some definite visual regressions between v1 and v2 but they’re already in v2 so I will open a migration review once we wrap up the accessibility improveents.

**Comment by davidmenendez** [comment]
> There are some definite visual regressions between v1 and v2 but they’re already in v2 so I will open a migration review once we wrap up the accessibility improveents.

yeah absolutely. this PR was opened prior to the greater accessibility inspection. i think we can just get this in for now to alleviate some temporary headaches, but most of this will be revisited in the follow up PR for the treegrid refactor.

**Review by elycheea — APPROVED** [review]
Marking as approved since other accessibility fixes are expected in another PR.

**Pull request merged**